client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -315,6 +374,19 @@ class CampaignAdminList extends SilverStripeComponent {
     return groups;
   }
 
+  getPlaceholderGroups() {
```

Comment:
ok, @phalkunz has explained why this is necessary. The only change I'd request is the switch to using the existing config for important_classes instead.